Dockerfile

```diff
@@ -79,7 +79,8 @@ RUN git clone --recursive /tmp/lightning . && \
 
 ARG DEVELOPER=0
 ENV PYTHON_VERSION=3
-RUN pip3 install mrkd
+RUN pip3 install mrkd mistune==0.8.4
```

Comment:
nit: Inside the docker, I think to use the requirements.lock it is better, what do you think?

Reply:
I'm not able to catch it :)
This one works well. I've just added like #4965

Reply:
Mh, I think in the docs it is fine, how mentioned in #4965 I didn't update the docker file because we hope that the package will be fixed in the next couple of weeks. If this happens, this docker file need another update,

If you use the following command `pip install -r requirements.lock` we can update the requirements.lock only, that it is tested to each commit in the main repository and I considered it well tested.